FileManager/Core/Services/Commands.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace FileManager.Core.Services
+{
+    [Flags]
+    public enum Commands
+    {
+        back, //переход к родительской директории
+        btr, //переход к главному каталогу
+        cd, //переход к поддиректории
+        clr, //чистка консоли
+        cp, //копирование файла/папки
+        crt, //создание файла/папки
+        dir, //показать список файлов и папок текущей директории
+        disk, //показать список дисков системы
+        diskpart, //переход к диску
+        diskreport, //отчет по дискам системы
+        exit, //выход из программы
+        help,
+        info,
+        move, //перемещение файла/папки
+        rename, //переименование файла/папки
+        rm, //удаление файлов/папок
+        size //размер файла/папки
+    }
+}
```

Comment:
Если это флаги, то каждому варианту надо указать его битовое значение. И комментарии надо писать системным образом.
```suggestion
    [Flags]
    public enum Commands
    {
        ///<summary>переход к родительской директории</summary>
        back = 0x0000001,
        btr = 0x0000002, //переход к главному каталогу
        cd = 0x0000004, //переход к поддиректории
        clr = 0x0000008, //чистка консоли
        cp = 0x0000010, //копирование файла/папки
        crt = 0x0000020, //создание файла/папки
        dir = 0x0000040, //показать список файлов и папок текущей директории
        disk = 0x0000080, //показать список дисков системы
        diskpart = 0x0000100, //переход к диску
        diskreport = 0x0000200, //отчет по дискам системы
        exit = 0x0000400, //выход из программы
        help = 0x0000800,
        info = 0x0001000,
        move = 0x0002000, //перемещение файла/папки
        rename = 0x0004000, //переименование файла/папки
        rm = 0x0008000, //удаление файлов/папок
        size = 0x0010000 //размер файла/папки
    }
}
```